tests/docker_run_tests.rs

```diff
@@ -238,9 +238,9 @@ fn test_node_custom_version() {
 
 #[test]
 fn test_node_no_lockfile() {
-    let name = simple_build("./examples/node-no-lockfile");
+    let name = simple_build("./examples/node-no-lockfile-canvas");
     let output = run_image(name, None);
-    assert!(output.contains("Hello from Node"));
+    assert!(output.contains("Hello from Node canvas"));
```

Comment:
```suggestion
    assert_eq!(output, "Hello from Node canvas");
```

Reply:
`npm run start` is used as start command here and that logs other stuff too.